lib/index.js

```diff
@@ -33,7 +33,9 @@ function swaggerize(server, options) {
  * @returns {onmount}
  */
 function mount(server, options) {
-    Object.defineProperty(server, 'swagger', {
+    if(!options.version)
+        options.version = 'v1';
+    Object.defineProperty(server, 'swagger'+options.version, {
```

Comment:
`server.swagger` (or rather `server.swagger.api.host`) is used to specify host that `swagger-ui` should send its requests to.

````javascript
server.listen(port, 'localhost', function () {
    app.swagger.api.host = server.address().address + ':' + server.address().port;
});
````

(From `swaggerize-express` README)